jsonschema/_utils.py

```diff
@@ -38,6 +38,22 @@ def __repr__(self):
         return repr(self.store)
 
 
+class Cache(object):
```

Comment:
We're making the cache a parameter though right? So you should be able to drop in something faster if the 2ms are relevant for your app?